### PR TITLE
[Fixes #3] Specs for Redis adapter

### LINKED ISSUE #3: Add specs for redis adapter

### PR DESCRIPTION
**Pull request by smakagon**

### CODE REVIEW

File: spec/unit/hanami/events/adapter/memory_spec.rb [hunk 1]
```diff
@@ -1,15 +1,15 @@
 RSpec.describe Hanami::Events::Adapter::Memory do
   let(:adapter) { described_class.new }
 
-  describe '#subscribe' do
-    it 'pushes listener to listener list' do
+  context '#subscribe' do
```
Comment (davydovanton) [inline spec/unit/hanami/events/adapter/memory_spec.rb]: I prefer `#describe` for method names :)
http://www.betterspecs.org/#describe

File: spec/unit/hanami/events/adapter/redis_spec.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+      adapter.subscribe('user.created', &handler)
+      adapter.subscribe('user.updated', &handler)
+    end
+  end
+
+  context '#broadcast' do
+    let(:redis) { Redis.new }
+
+    it 'calls #broadcast method with proper params on redis' do
```
Comment (davydovanton) [inline spec/unit/hanami/events/adapter/redis_spec.rb]:
```ruby
context 'with proper params on redis' do
  it 'calls #broadcast method'
end
```

File: spec/unit/hanami/events/adapter/memory_spec.rb (hunk 1 above)
Comment (smakagon) [inline spec/unit/hanami/events/adapter/memory_spec.rb]: Should I follow betterspecs and use shared_examples when needed? :trollface: 
Just kidding, ok, changing context to describe right now.

File: spec/unit/hanami/events/adapter/redis_spec.rb (hunk 2 above)
Comment (smakagon) [inline spec/unit/hanami/events/adapter/redis_spec.rb]: Sorry, "proper params" is related to `#broadcast` call, not to redis. :)
Probably it's better to remove "on redis" part to make this description less ambiguous

Comment (davydovanton) [inline spec/unit/hanami/events/adapter/redis_spec.rb]: agree 👍

**Pull request merged**